Makefile

```diff
@@ -76,6 +76,30 @@ package-dashboard-rpm:
 
 	rm -rf ${PWD}/build
 
+### build rpm for apisix-openresty:
+.PHONY: package-apisix-openresty-centos7-rpm
+package-apisix-openresty-centos7-rpm:
+	chmod +x ./build-apisix-openresty-centos7.sh
+	./build-apisix-openresty-centos7.sh
+	fpm -f -s dir -t rpm \
+		-n apisix-openresty \
+		-a `uname -i` \
+		-v $(version) \
+		--iteration $(iteration) \
+		-d 'openssl-devel' \
+		-d 'pcre-devel' \
+		-d 'gcc' \
+		-d 'curl' \
+		--description 'OpenResty distributions that include the APISIX patch.' \
+		--license "ASL 2.0" \
+		-C /usr/local/openresty/ \
+		-p ${PWD}/output/ \
+		--url 'http://apisix.apache.org/' \
+		--conflicts openresty \
+		--config-files usr/lib/systemd/system/openresty.service \
```

Comment:
@moonming 
When we use a file from OpenResty's repo, how do we clarify the license?